doc/SensorFusion/SRS/srs_eng.md

```diff
@@ -0,0 +1,372 @@
+
+## 1.2 Product Scope
+The developed software acquires sensor data from the Zumo, integrates external camera-based position data (SpaceShipRadar/OpenCV) via MQTT, and fuses them into a robust estimate of position, orientation, and velocity.  
+The solution runs both in the Webots simulation environment and on real hardware.
+The data will be provided via MQTT to other systems.
+
+## 1.3 Definitions, Acronyms, and Abbreviations
+- **Zumo32U4** – Small tracked robot from Pololu with ATmega32U4 and onboard sensors.  
+- **ZumoComSystem (ZCS)** – ESP32-based extension board providing communication and additional processing power.  
+- **SSR (SpaceShipRadar)** – External computer vision system (OpenCV-based) running on PC/Laptop, providing absolute position data via MQTT.  
+- **IMU (Inertial Measurement Unit)** – LSM6DS33 sensor (accelerometer and gyroscope) integrated on the Zumo32U4.  
+- **Odometry** – Estimation of traveled distance and rotation based on wheel encoders.  
+- **MQTT** – Message Queuing Telemetry Transport, a lightweight publish/subscribe communication protocol.  
+- **Sensor Fusion** – Combination of multiple data sources (e.g., odometry, IMU, vision) into a consistent state estimate (e.g., via EKF).  
+- **Webots** – Open-source robot simulator used for proof-of-concept evaluation.  
+- **Pose** – Robot’s position and orientation in 2D space (x, y, yaw).  
+
+## 1.4 References
+
+- **SpaceShipRadar (SSR)** – OpenCV-based external localization system:  
+  <https://github.com/NewTec-GmbH/SpaceShipRadar>  
+
+- **Webots** – Open-source robot simulator:  
+  <https://www.cyberbotics.com/>  
+
+- **RadonUlzer Applications** – Firmware for Zumo32U4:  
+  <https://github.com/BlueAndi/RadonUlzer>  
+
+- **DroidControlShip** – Firmware for ESP32 (ZumoComSystem):  
+  <https://github.com/BlueAndi/DroidControlShip>  
+
+- **Pololu Zumo32U4 User Guide** – Hardware documentation:  
+  <https://www.pololu.com/docs/0J63/all>  
+
+- **PEP-8** – Style Guide for Python Code:  
+  <https://peps.python.org/pep-0008/>  
+
+## 1.5 Overview
+This document is structured into the following main sections:
+
+- **Section 1 – Introduction**  
+  Defines the purpose, scope, terminology, and references relevant to this project.  
+
+- **Section 2 – Overall Description**  
+  Provides the system context, the product perspective, major functions, and assumptions or dependencies.  
+
+- **Section 3 – Requirements**  
+  Specifies the functional and non-functional requirements.  
+  Subdivided into interface definitions, system features, quality attributes, standards compliance, and implementation constraints.  
+
+- **Section 4 – Verification**  
+  Describes the methods and environments used to verify and validate the requirements.  
+
+- **Section 5 – Appendices**  
+  Contains supporting information such as glossary, analysis models, and a list of open issues (TBD).  
+
+
+# 2. Overall Description
+## 2.1 Product Perspective
```

Comment:
Missing line between headings. Do you have the Markdown extension enabled?
Also, there shall only be one top-level heading in the document.
Please enable the extension for more linting